<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormInputStreamView</class>
 <widget class="QWidget" name="FormInputStreamView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>74</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_sensorName">
     <property name="text">
      <string>SensorName</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_sourceType"/>
   </item>
   <item>
    <widget class="QLabel" name="label_frequency">
     <property name="text">
      <string>0 Hz</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
